doc/tutorial/bindings/python3/index.rst

```diff
@@ -0,0 +1,11 @@
+Python3
+=======
+
```

Comment:
I think that there should be some introduction. Like what are we trying to show our users in these tutorials, short summary what they can find there...